### DISCUSSION TITLE
how do i save/download the file to my desktop!

### DISCUSSION THREAD

**Discussion by ShantanuXD** [phcode-dev/phoenix#862] [Q&A] [answered]
hi, 
i use brave browser and i would like to know how i could the save/download file on my desktop,
i noticed that when i right-click there are 2 options "save" and "save as", i thought these option would let me download my code, but when i click it nothing happens 
i hope u could help with this problem.

**Comment by abose** [comment]
Are you on the default project? If so Download option should be available in a few weeks as its still being implemented. 
But you can open any folder in your computer and then edit it with the file - open folder option. You can also copy the file from default project and paste it in another project in Phoenix to get the file locally.

**Comment by abose** [comment, marked as answer]
Download feature is now available in phcode.dev @ShantanuXD
